Package.swift

```diff
@@ -724,3 +735,8 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(path: "../swift-collections"),
     ]
 }
+
+// Enable building macros as dynamic libraries by default for bring-up.
+for target in package.targets.filter({ $0.type == .regular || $0.type == .test }) {
+    target.swiftSettings = (target.swiftSettings ?? []) + [ .define("BUILD_MACROS_AS_DYLIBS") ]
+}
```

Comment:
This essentially flips the logic for now since dylib macros are the only ones which can be tested end-to-end right now.